src/styles/blocks/catalog.scss

```diff
@@ -22,4 +22,8 @@
       margin-right: 48px;
     }
   }
+  &__card:hover {
+    transform: scale(120%);
+    color: #34568b;
```

Comment:
![image](https://user-images.githubusercontent.com/96409563/206279995-c8ae89df-ec04-42a3-9e87-1f0fc4bb0ce7.png)
Also, it would be great to create variables for colors